gradoop-flink/src/main/java/org/gradoop/flink/datagen/transactions/foodbroker/config/FoodBrokerConfig.java

```diff
@@ -85,19 +89,23 @@ public static FoodBrokerConfig fromFile(String path) throws
    * @return list of String
    */
   public List<String> getStringValuesFromFile(String fileName) {
-    List<String> values = null;
-    String adjectivesPath = null;
-    // get path relevant to the config file
-    Path parentDirectory = Paths.get(path).getParent();
-    if (parentDirectory != null) {
-      adjectivesPath = parentDirectory.toString() + "/" + fileName;
-    }
+    String value = "";
+
+    InputStream inputStream = this.getClass().getResourceAsStream("/foodbroker/" + fileName);
     try {
```

Comment:
can you use
```java
try (InputStream inputStream = this.getClass().getResourceAsStream("/foodbroker/" + fileName)) {
  return org.apache.commons.io.IOUtils.toString(inputStream, "UTF-8");
}
```
instead of this try-catch-finally block?